torch/csrc/jit/mobile/interpreter.h

```diff
@@ -28,6 +29,7 @@ struct Code {
   std::vector<std::function<void(Stack&)>> operators_;
   std::vector<c10::IValue> constants_;
   std::vector<c10::TypePtr> types_;
+  std::vector<Function*> functions_;
```

Comment:
Is it needed? Looks like they are not initialized/used

Reply:
Good question! This is the place where we insert upgrader functions, so I suppose you initialize this :) But if that's not the plan I can update.